app/views/users/show.html.erb

```diff
@@ -1 +1,11 @@
-<h1>User Details</h1>
\ No newline at end of file
+<div class="container my-3">
+  <%= render partial: 'shared/user', locals: { user: @user } %>
+  <%= render partial: 'shared/bio', locals: { user: @user } %>
+
+  <% @posts.each do |post| %>
```

Comment:
* Kindly use the `#recent_post` method to render the user's 3 most recent posts as shown in the wireframe 
![image](https://github.com/anderCM/blog-app/assets/52497246/c327a759-3e8b-43e2-8c3d-1013ee005968)